entry/src/main/js/default/pages/index/index.hml

```diff
@@ -0,0 +1,171 @@
+                    <image class="weather" src="{{weather}}"></image>
+                </div>
+            </div>
+            <text class="time-txt">{{time}}</text>
+            <text class="day-date">{{day_date}}</text>
+            <div class="temp-page">
+                <text class="temp">{{temp_red}}</text>
+                <div>
+                    <image class="page" src="{{scroll_inactive}}"/>
+                    <image class="page" src="{{scroll_inactive}}" style="image-fill: white;"/>
+                    <image class="page" src="{{scroll_inactive}}"/>
+                    <image class="page" src="{{scroll_inactive}}"/>
+                    <image class="page" src="{{scroll_inactive}}"/>
+
+                </div>
+            </div>
+        </div>
+        <div class="music-bg" style="background-image: {{music_bg}};flex-direction: column;">
+            <image class="dreams-bg" src="{{music_cover_image}}"/>
+            <text class="song-name">{{song_name}}</text>
+            <text class="song-genre">{{song_genre}}</text>
+            <div class="set1_sub">
+                <image class="shuffle" src="{{shuffle}}"/>
+                <image class="back" src="{{back}}"/>
+                <image class="play" src="{{play}}"/>
+                <image class="next" src="{{next}}"/>
+                <image class="range" src="{{range}}"/>
+            </div>
+        </div>
+        <div class="food-remaining" style="background-image: {{music_bg}};">
+            <text class="food-remain">{{food_remain}}</text>
+            <percentage-bar product_name="Vegetables" product_percent="{{ vegetable_percentage }}"
+                            percent_color="{{green_range}}"></percentage-bar>
+            <percentage-bar product_name="Fruits" product_percent="{{ fruits_percentage }}"
+                            percent_color="{{violet_range}}"></percentage-bar>
+            <percentage-bar product_name="Meat" product_percent="{{ meat_percentage }}"
+                            percent_color="{{orange_range}}"></percentage-bar>
+
+        </div>
+    </div>
+
+    <text class="storage-settings">{{storage_settings}}</text>
+    <div class="set2" style="flex-direction: row;">
+        <div class="storage-tab" style="background-image: {{storage_bg}}; flex-direction:column">
+            <div class="your-storage-markers" style="flex-direction: row;">
+                <text class="storage-txt-1">{{your_storage}}</text>
+                <div>
+                    <image class="view-icon" src="{{view_icon}}"></image>
+                    <text class="view-txt">{{view_text}}</text>
+                    <image class="add-icon" src="{{add_icon}}"></image>
+                    <text class="add-txt">{{add_text}}</text>
+                </div>
+            </div>
+            <div style="flex-direction: row; margin-top:30.35px">
+                <list style="flex-direction: row; padding-right: 15px;">
+                    <list-item for="{{st_items}}">
```

Comment:
Instead of using st_items,  check if we can use the return value of getRecipeList